build/syntax-highlight.js

```diff
@@ -29,6 +35,20 @@ const loadAllLanguages = lazy(() => {
     "toml",
     "md",
     "cs",
+    "jsx",
+    "typescript",
+    "diff",
+    "ini",
+    "ignore",
+    "less",
+    "regex",
+    "scss",
+    "sql",
+    "uri",
+    "wasm",
+    "yaml",
+    "pug",
+    "webidl",
```

Comment:
Like I said, given @wbamberg take on this, is it wise to keep adding these until we have a policy for what we want or don't want. When should anyone be displaying webidl in a doc?

Reply:
We have a few cases where we display `webidl`, and we should get rid of them (Some are non-standard features, and the webidl in MDN is the only "spec" of the feature).

The only exception (and I don't think it needs syntax coloring) is the meta-doc article I wrote about how to create an interface/method/property page from reading the webidl in the spec.

Reply:
Yes. This is the down side of supporting everything. People use things that they should not.

Reply:
OK, It is 

```suggestion
    // "webidl", // block list
```

Reply:
I added the comment that it is in block list so we wouldn't have to maintain a block list separately. But OK with removing.

> Do we already have an open issue for tracking the removal of those "few cases where we display webidl"?

@teoli2003 is there an issue?